fhir_extractor/src/main.py

```diff
@@ -104,6 +73,8 @@ def run_resource(resource_mapping, analysis, primary_key_values=None):
     for record in list_records_from_db:
         logger.debug("One record from extract")
         event = dict()
+        event = {}
+        event["batch_id"] = batch_id
```

Comment:
Just to be sure I didn't miss something: the batch_id isn't yet used, is it?

Reply:
Indeed, we need to discuss how we want to use it. For now it's just added to the event